corehq/apps/reports/filters/location.py

```diff
@@ -0,0 +1,63 @@
+from dimagi.utils.decorators.memoized import memoized
+from corehq.apps.locations.models import SQLLocation
+from .users import ExpandedMobileWorkerFilter
+from .api import EmwfOptionsView
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext_lazy
+from corehq import toggles
+
+
+class LocationGroupFilter(ExpandedMobileWorkerFilter):
```

Comment:
hey Cory, after speaking with Danny/Cal about this, it seems like subclassing is the only way currently. I'll put in a doc string explaining the new subclass LocationGroupFilter